database/seeders/DatabaseSeeder.php

```diff
@@ -12,13 +15,21 @@ class DatabaseSeeder extends Seeder
      *
      * @return void
      */
-    public function run()
-    {
-        // \App\Models\User::factory(10)->create();
+    public function run(){
```

Comment:
The content of this run function code needs to be in the CategorySeeder file and you need to call the CategorySeeder to run in the DatabaseSeeder file like we did in Lab 4 (suggestion code below).